README.md

```diff
@@ -1,49 +1,92 @@
 # pay-selfservice
 GOV.UK Pay Self Service admin tool (Node.js)
 
-## Running locally with `pay local`
+## Prerequisites
+* This requires the [Pay CLI](https://github.com/alphagov/pay-infra/tree/master/cli), which is not publicly available at present.
+* You have [set up your local development environment](https://pay-team-manual.cloudapps.digital/manual/setup-local-dev-environment.html)
+* Clone this repo locally.
 
-*This requires the [Pay CLI](https://github.com/alphagov/pay-infra/tree/master/cli), which is not publicly available at present*
+### Running
 
-### Prerequisites
-You have [set up your local development environment](https://pay-team-manual.cloudapps.digital/development-processes/setup-local-dev-environment/)
+Self service reads from environment variables that allow it to connect to backend services from any environment. There are scripts to create an environment file to point to different backends.
 
-### Steps to run locally
-* Clone repo to your `$WORKSPACE` eg `cd $WORKSPACE && git clone git@github.com:alphagov/pay-selfservice.git && cd pay-selfservice`
-* `npm install`
-* `npm run compile`
-* `pay local launch admin --local selfservice`
-* Go to: http://localhost:9400
+#### Tunnel - easiest method
+Tunneling into different test and staging environments allows you to test with real data.
```

Comment:
`real data`  -> `environment specific data` or something similar ? `real` could be confused with live 🤷‍♂️

Reply:
Done